src/eos-google-chrome-installer

```diff
@@ -81,6 +90,24 @@ class GoogleChromeInstaller:
         # of a GSetting key under com.endlessm.GoogleChromeInitialSetup.
         return self._check_configuration() or self._check_legacy_gsetting()
 
+    def _initial_setup_already_done(self):
+        if os.path.exists(self.STAMP_FILE_INITIAL_SETUP_DONE):
+            return True
+
+        legacy_stamp_file = os.path.expanduser(self.LEGACY_USER_CONFIG_STAMP_FILE)
+        if os.path.exists(legacy_stamp_file):
+            # This is a legacy scenario, so we touch the system-wide file and
+            # remove the old one, so is not considered on the next log-in.
+            self._touch_done_file()
+            try:
+                os.unlink(legacy_stamp_file)
```

Comment:
No, stricly speaking I don't. But I thought it would be a nice thing to do. That said, it's extra I/O we don't need, so I might remove it.